### ISSUE TITLE
Variance with or without finite-sample adjustment?

### ISSUE THREAD

**Issue by zeileis** [Hapfelmeier/coat#12]
I noticed that the package is not fully consistent about computing the variance or standard deviation in each group. In most places we use the variance with division by the sample size n, i.e., the maximum-likelihood estimator. This is what we need to use for `mob()` so that the parameter instability inference works. But in some places we apply the finite-sample adjustment and divide by n-1, i.e., the least-squares estimator. This is probably easier and more intuitive but not fully consistent with the maximum-likelihood inference.

Any opinions on this?

Currently, `coef()` and `plot()` etc. use the version without adjustment. But `autoplot()` uses the version with adjustment.

**Comment by Hapfelmeier** [comment]
I'd like to discuss this issue in a meeting. I'll e-mail you in a second.

**Comment by zeileis** [comment]
I've looked at the code a bit more now and have one additional question in follow-up to today's discussion:

Now that we decided that the parameters that are tested by MOB are not necessarily exactly the parameters that we show in `coef()`, `print()`, and `plot()`. Should we report the standard deviation or the variance? And what are the parameter names we should use?

I suggest we use "Bias" (mean difference) and "SD" (standard deviation, not variance, of the difference, by default OLS) as we did already in the `print()` output.
